apps/base-docs/docs/pages/chain/node-performance.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Node Performance
+slug: /node-performance
+description: Guides on how to improve syncing performance of Base node clients
+---
+
+import { Button } from 'vocs/components'
+
+# Node Performance
+
+## Hardware
+
+We recommend the following hardware configuration to run a Base mainnet node:
+
+1.  A modern multi-core CPU with good single-core performance
+2.  At least 16 GB RAM (32 GB recommended)
+3.  A locally attached NVMe SSD drive
+4.  Adequate storage capacity to accommodate both the snapshot restoration process (if restoring from snapshot) and chain data, plus a 20% buffer for growth.
+
+:::info
+
+If utilizing Amazon Elastic Block Store (EBS), io2 Block Express is recommended to ensure timing buffered disk reads are fast enough in order to avoid latency issues alongside the rate of new blocks added to Base during the initial synchronization process.
+
+**The Base team highly recommends using locally attached NVMe SSD drives rather than networked storage.**
+
+:::
+
+## Client Software
+
+The [Base Node](https://github.com/base/node) repository contains the current stable configurations.
+
+#### Supported Clients
+
+Reth is currently the most performant client for running Base nodes. Future optimizations will be made
+primarily to Reth. You can read more about our migration to Reth [here](https://blog.base.dev/scaling-base-with-reth)
+
+| Type    | Supported Clients                                                                                        |
+| :------ | :------------------------------------------------------------------------------------------------------- |
+| Full    | [Reth](https://github.com/base/node/tree/main/reth), [Geth](https://github.com/base/node/tree/main/geth) |
+| Archive | [Reth](https://github.com/base/node/tree/main/reth)                                                      |
+
+## Geth LevelDB Tuning
```

Comment:
@0x00101010 do you think this belongs in a the new `Node Performance` docs page? I think some teams don't use base/node repo so this will help them.

Reply:
FWIW, also agree this would be helpful to have here 👍